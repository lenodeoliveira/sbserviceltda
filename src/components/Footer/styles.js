import styled, { css } from 'styled-components'
import { Container as SectionContainer } from '../SectionContainer'

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align: center;
    border-top: 0.1rem solid ${theme.colors.mediumGray};
    p {
      font-size: 1.8rem;
    }
    & ${SectionContainer} {
      padding-top: 0;
      padding-bottom: 0;
    }
  `}
`
