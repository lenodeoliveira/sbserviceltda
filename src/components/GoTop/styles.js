import styled, { css } from 'styled-components'

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
    color: ${theme.colors.white};
    background: ${theme.colors.primaryColor};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    opacity: 0.8;
  `}
`
