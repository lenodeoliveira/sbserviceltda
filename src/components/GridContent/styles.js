import styled, { css } from 'styled-components'

export const Container = styled.div`
  ${() => css`
    text-align: center;
    max-width: 58rem;
    margin: 0 auto;
  `}
`
export const TextContainer = styled.div`
  ${({ theme }) => css`
    margin: ${theme.spacings.xhuge} 0;
  `}
`
