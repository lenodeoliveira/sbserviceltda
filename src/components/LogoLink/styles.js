import styled, { css } from 'styled-components'

export const Container = styled.a`
  ${() => css`
    text-decoration: none;
    display: flex;
    align-items: center;
    color: inherit;
    > img {
      max-height: 2.5rem;
    }
  `}
`
