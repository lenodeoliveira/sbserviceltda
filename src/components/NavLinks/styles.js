import styled, { css } from 'styled-components'

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-flow: row wrap;
    background: red;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`
