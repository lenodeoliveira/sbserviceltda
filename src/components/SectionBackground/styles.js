import styled, { css } from 'styled-components'

const backgroundActivate = (theme) => css`
  background: ${theme.colors.primaryColor};
  color: ${theme.colors.white};
`

export const Container = styled.div`
  ${({ theme, background }) => css`
    background: ${theme.colors.white};
    color: ${theme.colors.primaryColor};
    ${background && backgroundActivate(theme)};
    min-height: 100vh;
    display: flex;
    align-items: center;
  `}
`
