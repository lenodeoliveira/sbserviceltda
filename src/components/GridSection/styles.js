import styled, { css } from 'styled-components'
import { Title as HeadingContainer } from '../Heading/styles'
import { Container as TextComponent } from '../TextComponent/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    ${TextComponent} {
      margin-bottom: ${theme.spacings.xhuge};
    }
  `}
`
export const Grid = styled.div`
  ${({ theme }) => css`
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
    gap: ${theme.spacings.medium};
  `}
`
export const GridElement = styled.div`
  ${() => css`
    ${HeadingContainer} {
    }
  `}
`
