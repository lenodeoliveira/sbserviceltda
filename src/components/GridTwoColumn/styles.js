import styled, { css } from 'styled-components'
import { Title } from '../Heading/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    display: grid;
    grid-template-columns: 1.8fr 1fr;
    margin-left: 3rem;
    align-items: center;
    gap: ${theme.spacings.large};

    @media ${theme.media.lteMedium} {
      grid-template-columns: 1fr;
      text-align: center;
      margin-left: 0;
    }
    ${Title} {
      margin-bottom: ${theme.spacings.xlarge};
    }
  `}
`
export const TextContainer = styled.div`
  ${({ theme }) => css`
    @media ${theme.media.lteMedium} {
      margin-bottom: ${theme.spacings.large};
    }
  `}
`
export const ImageContainer = styled.div`
  ${() => css`
    width: 100%;
  `}
`
export const Image = styled.img`
  ${() => css`
    width: 100%;
    border-radius: 2.4%;
  `}
`
