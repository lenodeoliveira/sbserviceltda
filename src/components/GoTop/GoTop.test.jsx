import { screen } from '@testing-library/react'
import { GoTop } from '.'
import { renderTheme } from '../../styles/render-theme'

describe('<GoTop />', () => {
  it('Should render', () => {
    const { container } = renderTheme(<GoTop>Children</GoTop>)
    expect(screen.getByRole('link', { name: 'Go to top' })).toBeInTheDocument()
    expect(screen.getByRole('link', { name: 'Go to top' })).toHaveAttribute('href', '#')
    expect(container).toMatchSnapshot()
  })
})
