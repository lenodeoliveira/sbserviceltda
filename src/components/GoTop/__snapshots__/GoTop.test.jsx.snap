// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GoTop /> Should render 1`] = `
.c0 {
  position: fixed;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
  color: #FFFFFF;
  background: #2F5233;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  opacity: 0.8;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

<div>
  <a
    aria-label="Go to top"
    class="c0"
    href="#"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
